### ISSUE TITLE
eslint prefer static class properties

### ISSUE THREAD

**Issue by graingert** [eslint/eslint#9029]
**Please describe what the rule should do:**
Move static class properties into the class body.

**What category of rule is this? (place an "X" next to just one item)**

[x] Enforces code style
[ ] Warns about a potential error
[x] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js
// bad
class Foo {}
Foo.bar = 3;

// good

class Foo {
  static bar = 3;
}
```

**Why should this rule be included in ESLint (instead of a plugin)?**
It's pretty simple imhop.

**Comment by not-an-aardvark** [comment]
Thanks for the proposal. However, class properties are experimental syntax, which we don't support (see our [readme](https://github.com/eslint/eslint/blob/0f9727902fce753c87f45d439c521c93850d7dd8/README.md#what-about-experimental-features)).

**Comment by graingert** [comment]
Oh whoops, for some reason I thought they weren't experimental

**Comment by platinumazure** [comment]
Definitely feel free to create an issue in eslint-plugin-babel though!

**Comment by graingert** [comment]
@platinumazure done: https://github.com/babel/eslint-plugin-babel/issues/128 thanks for the suggestion.
